### PR TITLE
add cluster name in analysis log

### PR DESCRIPTION
**Pull request by yangeagle**
ref: https://github.com/openark/orchestrator/issues/1125

### CODE REVIEW

File: go/inst/instance_dao.go [hunk 1]
```diff
@@ -301,6 +301,8 @@ func ReadTopologyInstanceBufferable(instanceKey *InstanceKey, bufferWrites bool,
 	errorChan := make(chan error, 32)
 	var resolveErr error
 
+	instance.Key = *instanceKey
+
```
Comment (shlomi-noach) [inline go/inst/instance_dao.go]: Can you please clarify why you moved this line to this location? `if !instanceKey.IsValid()` is important context. If the key is invalid, we do not assign `instance.Key`.

**Review by shlomi-noach — COMMENTED** [review]
Sorry for the late response. Adding cluster name is 👍 . Asking to clarify what seems to be an unrelated change.

**Comment by yangeagle** [comment]
> Sorry for the late response. Adding cluster name is 👍 . Asking to clarify what seems to be an unrelated change.

Sorry for extra change. I have fixed it
@shlomi-noach

**Comment by shlomi-noach** [comment]
resubmitted as https://github.com/openark/orchestrator/pull/1152 because this PR originates at `master` branch

**Pull request merged**